src/redux/actions/articles.js

```diff
@@ -72,3 +94,43 @@ export const createArticle = articleData => async (dispatch) => {
     return error.response.data;
   }
 };
+
+export const likeCurrentArticle = slug => async (dispatch) => {
+  try {
+    const token = localStorage.getItem('token');
+    const { data } = await axios.post(
+      `${apiUrl}/like_article/${slug}`,
+      {},
```

Comment:
@youngestdj what's does this empty object do?

Reply:
@9jaswag the post method expects data to be sent to the URL in the second argument. We are not sending any data so the object is empty.

Reply:
Please fix this too